Move test model folders

ydshieh opened the pull request:
# What does this PR do?

Move test model folders

ydshieh committed:
move test model folders (TODO: fix imports and others)

ydshieh committed:
fix (potentially partially) imports (in model test modules)

ydshieh committed:
fix (potentially partially) imports (in tokenization test modules)

ydshieh committed:
fix (potentially partially) imports (in feature extraction test modules)

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_16945). All of your documentation changes will be reflected on that endpoint.

ydshieh committed:
fix import utils.test_modeling_tf_core

ydshieh committed:
fix path ../fixtures/

ydshieh committed:
fix imports about generation.test_generation_flax_utils

ydshieh committed:
fix more imports

ydshieh committed:
fix fixture path

ydshieh committed:
fix get_test_dir

ydshieh committed:
update module_to_test_file

ydshieh committed:
fix get_tests_dir from wrong transformers.utils

ydshieh committed:
update config.yml (CircleCI)

ydshieh committed:
fix style

ydshieh committed:
remove missing imports

ydshieh committed:
update new model script

ydshieh committed:
update check_repo

ydshieh committed:
update SPECIAL_MODULE_TO_TEST_MAP

ydshieh committed:
fix style

ydshieh committed:
test - to be revert

ydshieh committed:
temp change to generate new cache - to be reverted

ydshieh committed:
temp change to generate new cache - to be reverted

ydshieh committed:
clean-up

ydshieh committed:
add __init__

ydshieh committed:
update self-scheduled

ydshieh committed:
fix add_new_model scripts

ydshieh committed:
check one way to get location back

ydshieh committed:
python setup.py build install

# Conflicts:
#	.github/workflows/add-model-like.yml
#	.github/workflows/model-templates.yml

ydshieh committed:
Run the scheduled tests via push

ydshieh committed:
update
